cookbook/getting-started/appbar-basics.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: page
+title: "AppBar Basics"
+---
+
+Todo: Rewrite this with more detail / less complex state management 
+
+-----
+
+A typcial AppBar with a title, actions, and an overflow dropdown menu.
```

Comment:
Typical misspelled

Reply:
Going to remove this article since it actually already resides in the Sample Catalog